support-frontend/assets/helpers/internationalisation/billableCountries.ts

```diff
@@ -0,0 +1,240 @@
+const billableCountries: Record<string, string> = {
```

Comment:
Is it worth using `IsoCountry` (from internationalisation/country) for this and the other similar types?

```suggestion
const billableCountries: Record<IsoCountry, string> = {
```

Reply:
Addressed.